tests/integration/targets/ios_vrf_interfaces/tests/common/rendered.yaml

```diff
@@ -0,0 +1,21 @@
+---
```

Comment:
Can we also include tests files for replaced and deleted

Reply:
done